spec/index.bs

```diff
@@ -7142,22 +7142,21 @@ dictionary GPURenderPassDepthStencilAttachment {
     Given a {{GPURenderPassDepthStencilAttachment}} |this| the following validation
     rules apply:
 
-    - |this|.{{GPURenderPassDepthStencilAttachment/view}} must have a [=depth or stencil renderable
+    1. |this|.{{GPURenderPassDepthStencilAttachment/view}} must have a [=depth or stencil renderable
         format=].
-    - |this|.{{GPURenderPassDepthStencilAttachment/view}} must be a view of a
+    1. |this|.{{GPURenderPassDepthStencilAttachment/view}} must be a view of a
         single [=texture subresource=].
-    - |this|.{{GPURenderPassDepthStencilAttachment/view}}.{{GPUTexture/[[descriptor]]}}.{{GPUTextureDescriptor/usage}}
+    1. |this|.{{GPURenderPassDepthStencilAttachment/view}}.{{GPUTexture/[[descriptor]]}}.{{GPUTextureDescriptor/usage}}
         must contain {{GPUTextureUsage/RENDER_ATTACHMENT}}.
-    - If |this|.{{GPURenderPassDepthStencilAttachment/view}}.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/format}}
+    1. If |this|.{{GPURenderPassDepthStencilAttachment/view}}.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/format}}
         contains both depth and stencil aspects (See [[#depth-formats|depth-stencil formats]]):
-        - |this|.{{GPURenderPassDepthStencilAttachment/depthReadOnly}} must be equal to |this|.{{GPURenderPassDepthStencilAttachment/stencilReadOnly}}
-    - If |this|.{{GPURenderPassDepthStencilAttachment/depthReadOnly}} is `true`:
-        - |this|.{{GPURenderPassDepthStencilAttachment/depthLoadValue}} must be {{GPULoadOp/"load"}}.
-        - |this|.{{GPURenderPassDepthStencilAttachment/depthStoreOp}} must be {{GPUStoreOp/"store"}}.
-    - If |this|.{{GPURenderPassDepthStencilAttachment/stencilReadOnly}} is `true`,
-        |this|.{{GPURenderPassDepthStencilAttachment/stencilLoadValue}} must be
-        {{GPULoadOp/"load"}} and |this|.{{GPURenderPassDepthStencilAttachment/stencilStoreOp}}
-        must be {{GPUStoreOp/"store"}}.
+        1. |this|.{{GPURenderPassDepthStencilAttachment/depthReadOnly}} must be equal to |this|.{{GPURenderPassDepthStencilAttachment/stencilReadOnly}}
+    1. If |this|.{{GPURenderPassDepthStencilAttachment/depthReadOnly}} is `true`:
+        1. |this|.{{GPURenderPassDepthStencilAttachment/depthLoadValue}} must be {{GPULoadOp/"load"}}.
+        1. |this|.{{GPURenderPassDepthStencilAttachment/depthStoreOp}} must be {{GPUStoreOp/"store"}}.
```

Comment:
Do we need to add "|this|.{{GPURenderPassDepthStencilAttachment/view}}.{{GPUTextureView/[[descriptor]]}}.{{GPUTextureViewDescriptor/format}} must contain depth aspect", and a similar one for stencil?

Reply:
good idea!

Reply:
Done.

Reply:
If I'm reading correctly this is a new validation rule, right? @kvark I don't think this really makes sense. You could just as easily specify `depthReadOnly: false` explicitly, why should that be allowed but `depthReadOnly: true` be disallowed?

I don't think this needs a validation rule unless it's adding something I don't understand.

Reply:
If `depthReadOnly` is `false`, maybe some combinations with `depthStoreOp` and `depthLoadValue` also require format contain depth aspect? I mean, the textureView attached should be explicitly required to contain depth aspect when it is definitely going to operate on depth aspect for some combinations via setting these values. The question is: what are the combinations? If `depthReadOnly` is explicitly set to true, I am not 100% sure, but my understanding is that we are going to read data from depth aspect from depth/stencil attachment. It is similar for stencil aspect. WDYT, Kai?

Reply:
`depthReadOnly: true` only prevents writing, it has no bearing on whether depth is read or not. (`setPipeline` [validates](https://gpuweb.github.io/gpuweb/#dom-gpurenderencoderbase-setpipeline) the pipeline against the `depthReadOnly`/`stencilReadOnly` of the pass.) It's the "this.[[layout]] equals pipelineTargetsLayout" rule that validates a pipeline's (color and depth) attachments against the render pass. That requires the exact formats to match. And #2079 requires the GPUTextureView to reference all aspects.

I don't think createRenderPipeline has any validation to ensure depth-writing flags aren't used with depth-less stencil formats, but I think it's implicit in the pipeline creation that it won't do anything?

Reply:
OK. It makes sense. Thanks for the details. 

I removed that incorrect validation rules. Now the PR just correct some typos and style stuff. Please help to merge it if it is OK, @kainino0x .

Reply:
LGTM. @kvark please also review this thread in case there's some edit you still want to make to this validation logic.